packages/website/components/Sidebar.tsx

```diff
@@ -106,13 +106,12 @@ export function Sidebar({ currentPage = '/' }: Props) {
           >
             Contributing to Forma 36
           </SidebarLink>
-          <SidebarLink
-            isActive={isLinkActive('/migration-v3-to-v4', currentPage)}
-            href="/migration-v3-to-v4"
-          >
-            Migration Guide
-          </SidebarLink>
           <Box marginBottom="spacingL" />
```

Comment:
Uhhhh very unrelated to this PR, but isn't this Box creating an empty `div` in the website just to add a margin bottom?

Reply:
hahaha, seems like it :D